### DISCUSSION TITLE
Problem installing version 5.3 - Windows 11

### DISCUSSION THREAD

**Discussion by Benlge** [SamDel/ChromeCast-Desktop-Audio-Streamer#153] [Q&A] [answered]
Hi,

Hello to the community.

I have an installation problem when I want to install both ChromeCast-Desktop-Audio-Streamer-5.3 and Setup.5.3.
I am attaching the error video screenshot that prevents me from installing this version on Windows 11.

Best regards

Ben

https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/assets/64033346/091cb66a-dec6-4a0a-b3c4-ac5cb6156542

**Comment by SamDel** [comment, marked as answer]
Hi Ben,

I haven't seen that one before. 
Does the solution on [this page](https://stackoverflow.com/questions/781402/msi-install-error-this-advertised-application-will-not-be-installed-because-it) :

``` 
msiexec /i Setup.msi /qb
```

work for you?

**Reply by Benlge** [reply]
I will try, thank you.

**Reply by Benlge** [reply]
It works! Thank you so much !!!

**Reply by SamDel** [reply]
Glad it works.
I'm not sure if I can change something to prevent these errors for other users.

I've added [a line about it in the wiki](https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/wiki#error-when-running-the-installer-this-advertised-application-will-not-be-installed-).

**Reply by Benlge** [reply]
I think it comes from me. I just repaired, recovered, all my data in Windows 11. I just saw this problem for the first time. I've been using this program for years, and never had this problem, even with Windows 11. Think it's an isolated problem. Thank you so much. Best regards. Ben
